src/MainUpdate.elm

```diff
@@ -336,6 +336,7 @@ update msg model =
                     invalidSubmissionIds =
                         newSubmissionIdsString
                             |> split ","
+                            |> List.map (\sub -> String.trim sub)
```

Comment:
no need for anonymous function here. Can be ` List.map String.trim`. Doesn't need to be changed but maybe useful for future PRs